pages/common/czkawka-cli.md

```diff
@@ -0,0 +1,16 @@
+# czkawka-cli
+
+> Multi functional app to find duplicates, empty folders, similar images etc. Command-line version.
+> More information: <https://github.com/qarmin/czkawka>.
+
+- Find [dup]licate files in directory:
+
+`czkawka-cli dup --directories {{path/to/directory}}`
+
+- Find [dup]licate files and delete them (default: `NONE`):
+
+`czkawka-cli dup --directories {{path/to/directory}} --delete-method {{AEN/AEO/ON/OO/HARD/NONE}}`
```

Comment:
```suggestion
- Find duplicate files in specific directories and delete them (default: `NONE`):

`czkawka-cli dup --directories {{path/to/directory1 path/to/directory2 ...}} --delete-method {{AEN|AEO|ON|OO|HARD|NONE}}`
```

- Optional fix: example can be generalized.
- Mandatory fix: [`/` must be replaced with `|`](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).